org.tropy.Tropy.yml

```diff
@@ -13,14 +13,16 @@ finish-args:
   #- --socket=wayland
   #- --socket=fallback-x11
   - --socket=x11
-  - --socket=pulseaudio
+  #- --socket=pulseaudio
   - --share=network
   - --device=dri
   - --filesystem=home
   - --filesystem=/media
   - --filesystem=/run/media
+  - --filesystem=/tmp
```

Comment:
Why do you need access to real temp? Can setting the TMPDIR variable work?

Reply:
Yes probably so. I had issues with the Chromium lockfiles and thought that this fixed it at some point.